install/resources/upgrade41.php

```diff
@@ -35,6 +35,21 @@ function upgrade41_dbchanges()
 	$db->insert_query('spiders', array("name" => "DuckDuckGo", "useragent" => "DuckDuckBot/1.1; (+http://duckduckgo.com/duckduckbot.html)"));
 	$db->insert_query('spiders', array("name" => "UptimeRobot", "useragent" => "Mozilla/5.0+(compatible; UptimeRobot/2.0; http://www.uptimerobot.com/)"));
 
-  	$output->print_contents("<p>Click next to continue with the upgrade process.</p>");
+	// Remove backslashes from last 1,000 log files.
+	$query = $db->simple_select('moderatorlog', '*', "action LIKE '%\\\\\\%", "limit" => 1000);
```

Comment:
If all we're using is `tid` and `action`, no need to `SELECT *`, use `$db->simple_select('moderatorlog', 'tid, action', "action LIKE '%\\\\\\%", "limit" => 1000);` instead.

Also should order by `tid` descending.

Reply:
I'm making those adjustments now.